### PR TITLE
Updating RStudio links.

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Updating RStudio note links with the full URL.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by jiridanek — APPROVED** [review]
just that i've seen this and have no objections; afaik there's a designated doc team approver for these things who has the final word

**Review by chtyler — APPROVED** [review]

**Pull request merged**
